<launch>

  <arg name="use_arm" default="both" />
  
  <!-- Joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="arm_teleop_joy" >
    <param name="dev" type="string" value="/dev/input/js1" />
    <param name="deadzone" value="0.5" />
  </node>

  <!--   
  <node pkg="bender_arm_control" type="arm_teleop" name="arm_teleop">
    <rosparam param="arm_name">["l_arm", "r_arm"]</rosparam>
    <param name="rate" type="int" value="25" />
  </node>
  -->

  <!-- Launch robot model -->
  <include file="$(find bender_tf)/launch/tf_model.launch"/>

  <!-- Launch selected arm -->
  <include file="$(find bender_arm_control)/launch/$(arg use_arm).launch"/>
    
  <!-- MoveIt -->
  <include file="$(find bender_arm_control)/launch/planning.launch" >
    <arg name="use_arm" value="$(arg use_arm)" />
  </include>

  <!-- Arm teleop -->
  <node pkg="bender_arm_control" type="arm_teleop.py" name="simple_arm_teleop" output="screen">
    <param name="use_arm" value="$(arg use_arm)"/>
    <param name="rate" type="int" value="5" />
  </node>

</launch>